examples/whisper.py

```diff
@@ -297,7 +306,10 @@ def listener(q):
   print("done listening")
 
 if __name__ == "__main__":
-  model, enc = init_whisper("small.en" if getenv("SMALL") else "tiny.en", batch_size=1)
+  # Set WHISPER_MODEL to change the inference model.
+  # SMALL is for compat with tinygrad<=0.10.2.
+  model_name = getenv("WHISPER_MODEL", default="small.en" if getenv("SMALL") else "tiny.en")
```

Comment:
Why is this an env var?

Reply:
The existing code has been using an environment variable (`SMALL`)
to switch between models. (It only supported `small.en` and `tiny.en`
for unknown reasons, though).

My intention here is to follow that convention.

Reply:
_Addendum:_ I can make it a command-line argument if you prefer.
In particular, I'm thinking something like this:

* https://github.com/tinygrad/tinygrad/blob/master/examples/gpt2.py#L204-L214
* https://github.com/tinygrad/tinygrad/blob/master/examples/llama.py#L333-L344